api/app/controllers/spree/api/v2/storefront/cart_controller.rb

```diff
@@ -50,11 +50,25 @@ def remove_line_item
           def empty
             spree_authorize! :update, spree_current_order, order_token
 
-            # TODO: we should extract this logic into service and let
-            # developers overwrite it
-            spree_current_order.empty!
+            result = spree_current_order.empty!
```

Comment:
@KacperMekarski let's not call `empty!` here - let's use the service directly, also the return value of `empty!` should be the the order itself, the same as before, we cannot break this behaviour

Reply:
done.